Gemfile.lock

```diff
@@ -265,14 +288,14 @@ DEPENDENCIES
   ims-lti (= 1.1.13)
```

Comment:
ims-lti has a few newer versions, which might be candidates to change now that rails is updated.

Reply:
The change is of course to the gemfile, not this file

Reply:
Ok, but let's do this a bit later, after we have verified that Rubyric indeed works after these changes?